src/transformers/generation/configuration_utils.py

```diff
@@ -189,6 +189,15 @@ class GenerationConfig(PushToHubMixin):
             The guidance scale for classifier free guidance (CFG). CFG is enabled by setting `guidance_scale > 1`.
             Higher guidance scale encourages the model to generate samples that are more closely linked to the input
             prompt, usually at the expense of poorer quality.
+        negative_prompt (`torch.LongTensor` of shape `(batch_size, sequence_length)`, *optional*):
+            The negative prompt to use for CFG. Will throw an error if set but `cfg_scale` <= 1 or None. The batch size
+            must match the input batch size. If unset, it will be defaulted to the last input token.
+        negative_prompt_attention_mask (`torch.LongTensor` of shape `(batch_size, sequence_length)`, *optional*):
+            Attention_mask for `negative_prompt`.
```

Comment:
@Vermeille these two arguments are not part of the method configuration (similarly to the main prompt and its corresponding attention mask), so they have to be part of the signature of `generate` and passed into `_get_logits_processor`.

Should be an easy change 🤗